content/en/docs/ambient/usage/policy/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Policy enforcement in ambient mode
+description: The two enforcement points for policy in an ambient mesh.
+weight: 20
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+Istio's ambient data plane splits the data plane into two different sets of components. This architecture allows users to only pay for application layer processing if they need it. The trade-off made is the extra complexity of understanding how traffic flows through this architecture and where policy will be enforced. This guide will introduce a number of broad scenarios and explore the permutations therein and the details to understand for each scenario.
```

Comment:
`extra complexity` - we need to spin positively. flexibility, pay only what you need

Reply:
I think we just delete this page. Right now it is basically just storing notes.

Reply:
Ok, maybe good to have dedicated page for authz policy.